inc/patterns/general-404.php

```diff
@@ -15,4 +15,5 @@
 					<!-- wp:spacer {"height":128} -->
 					<div style="height:128px" aria-hidden="true" class="wp-block-spacer"></div>
 					<!-- /wp:spacer -->',
+	'inserter'   => false,
```

Comment:
Really minor, but can we move this up above the content? I think it's less likely to be missed that way. Also, do we still need the category attribute?

Reply:
> Also, do we still need the category attribute?

Nope!

> can we move this up above the content?

👍